specification/2.0/README.md

```diff
@@ -2086,14 +2023,14 @@ A view into a buffer generally representing a subset of the buffer.
 
 |   |Type|Description|Required|
 |---|----|-----------|--------|
-|**buffer**|`integer`|The index of the buffer.| :white_check_mark: Yes|
+|**buffer**|[`glTFid`](#reference-gltfid)|The index of the buffer.| :white_check_mark: Yes|
 |**byteOffset**|`integer`|The offset into the buffer in bytes.|No, default: `0`|
-|**byteLength**|`integer`|The length of the bufferView in bytes.| :white_check_mark: Yes|
+|**byteLength**|`integer`|The total byte length of the buffer view.| :white_check_mark: Yes|
```

Comment:
I think the reference has better wording than the schema here, we should copy it down.